src/main/webapp/placeGuides/PlaceGuideOnList.js

```diff
@@ -36,6 +37,44 @@ class PlaceGuideOnList {
     PlaceGuideOnList.close(placeGuideId);
   }
 
+  static expand(placeGuideId) {
+    const divId = 'placeGuideOnList-' + '{' + placeGuideId + '}';
+    const placeGuideDiv = document.getElementById(divId);
+    placeGuideDiv.querySelectorAll('.folded-placeGuide')[0]
+        .style.display = 'none';
+    placeGuideDiv.querySelectorAll('.card-placeGuide')[0]
+        .style.display = 'block';
+    placeGuideDiv.style.padding = '0px';
```

Comment:
I'm a bit confused. If it is supposed to expand it, why is it hiding it? Or is it just me not getting CSS? 😅

Reply:
The folded-placeguide and the card-placeguide are different elements, and only one of them is visible all the time. If the card is closed, then folded-placeguide is shown, if the card is expanded, then card-placeguide. I think this explains what happens here. 
Remark that this is not my code, I might have changed the order of the functions only and it appears as if I wrote it.